src/main/java/picard/arrays/MergePedIntoVcf.java

```diff
@@ -0,0 +1,260 @@
+import htsjdk.variant.variantcontext.writer.VariantContextWriterBuilder;
+import htsjdk.variant.vcf.VCFEncoder;
+import htsjdk.variant.vcf.VCFFileReader;
+import htsjdk.variant.vcf.VCFFormatHeaderLine;
+import htsjdk.variant.vcf.VCFHeader;
+import htsjdk.variant.vcf.VCFHeaderLine;
+import htsjdk.variant.vcf.VCFHeaderLineType;
+import htsjdk.variant.vcf.VCFInfoHeaderLine;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import picard.PicardException;
+import picard.cmdline.CommandLineProgram;
+import picard.cmdline.StandardOptionDefinitions;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static htsjdk.variant.variantcontext.Allele.SPAN_DEL_STRING;
+import static htsjdk.variant.vcf.VCFConstants.MISSING_VALUE_v4;
+import static htsjdk.variant.vcf.VCFConstants.UNPHASED;
+import static picard.arrays.MergePedIntoVcf.USAGE_DETAILS;
+import static picard.arrays.illumina.InfiniumVcfFields.ALLELE_A;
+import static picard.arrays.illumina.InfiniumVcfFields.ALLELE_B;
+
+/**
+ * Class to take genotype calls from a ped file output from zCall and merge them into a vcf from autocall.
+ */
+@CommandLineProgramProperties(
+        summary = USAGE_DETAILS,
+        oneLineSummary = "Program to merge a single-sample ped file from zCall into a single-sample VCF.",
+        programGroup = picard.cmdline.programgroups.GenotypingArraysProgramGroup.class
+)
+public class MergePedIntoVcf extends CommandLineProgram {
+
+    static final String USAGE_DETAILS = "MergePedIntoVcf takes a single-sample ped file output from zCall and merges " +
+            "into a single-sample vcf file using several supporting files." +
+            "A VCF, aka Variant Calling Format, is a text file for storing how a sequenced sample differs from the reference genome. " +
+            "<a href='https://samtools.github.io/hts-specs/VCFv4.2.pdf'></a>" +
+            "A PED file is a whitespace-separated text file for storing genotype information. " +
+            "<a href='http://zzz.bwh.harvard.edu/plink/data.shtml#ped'></a>" +
+            "A MAP file is a whitespace-separated text file for storing information about genetic distance. " +
+            "<a href='http://zzz.bwh.harvard.edu/plink/data.shtml#map'></a>" +
+            "A zCall thresholds file is a whitespace-separated text file for storing the thresholds for " +
+            "genotype clustering for a SNP as determined by zCall." +
+            "<a href='https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3463112/#SEC2title'></a>" +
+            "<h4>Usage example:</h4>" +
+            "<pre>" +
+            "java -jar picard.jar MergePedIntoVcf \\<br />" +
+            "VCF=input.vcf \\<br />" +
+            "PED=zcall.output.ped \\<br />" +
```

Comment:
The convention is to have the parameters inset here (see SortSam usage for an example).